wcfsetup/install/files/lib/system/user/command/RegistrationNotification.class.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace wcf\system\user\command;
+
+use wcf\data\user\User;
+use wcf\system\user\notification\object\UserRegistrationUserNotificationObject;
+use wcf\system\user\notification\UserNotificationHandler;
+use wcf\system\WCF;
+
+/**
+ * Send a notification of user registration status.
+ *
+ * @author      Olaf Braun
+ * @copyright   2001-2024 WoltLab GmbH
+ * @license     GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @since       6.1
+ */
+final class RegistrationNotification
```

Comment:
The name of commands should contain an action verb. For example `CreateRegistrationNotification`.